### PR TITLE
Remove normalize.css and Optimize head tag order.

### LINKED ISSUE #3033: Normalize.css still referenced in index.html
https://github.com/h5bp/html5-boilerplate/blob/87bdc80ab9e35dcb52d75ffc926d8ed67991c626/src/index.html#L19

Removed in #2960

### PR DESCRIPTION
**Pull request by rgant**
Remove normalize.css link. Closes #3033 

Optimize header tag order:
> How you order elements in the <head> can have an effect on the (perceived) performance of the page.
> This script helps you identify which elements are out of order.
https://github.com/rviscomi/capo.js

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **[CONTRIBUTING](https://github.com/h5bp/html5-boilerplate/blob/main/.github/CONTRIBUTING.md)** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

Pull requests should be thought of as a conversation. There will be some back and forth when trying to get code merged into this or any other project. With all but the simplest changes you can and should expect that the maintainers of the project will request changes to your code. Please be aware of that and check in after you open your PR in order to get your code merged in cleanly.

Thanks!

### CODE REVIEW

**Comment by rgant** [comment]
I agree to allow the project owners to license your work under the terms of the [MIT License](https://github.com/h5bp/html5-boilerplate/blob/main/.github/LICENSE.txt).

**Comment by coliff** [comment]
looks good - can you please make build so it updates dist/ ?

**Review by roblarsen — APPROVED** [review]
This looks great. Thanks! It saves a lot of work updating the docs.

File: package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
     "strip-json-comments": "^5.0.1"
   },
   "engines": {
-    "node": ">=14"
+    "node": ">=16",
+    "npm": ">=8"
```
Comment (rgant) [inline package.json:43]: The tests only [cover 16 & 18](https://github.com/h5bp/html5-boilerplate/blob/87bdc80ab9e35dcb52d75ffc926d8ed67991c626/.github/workflows/test.yaml#L17) so I assumed that should be the minimums for the engines specification.

**Comment by coliff** [comment]
Great PR - thanks again!

**Pull request merged**